packages/connect-wallet-modal/package.json

```diff
@@ -37,16 +37,21 @@
     "lint": "eslint --ext ts,tsx,js,mjs ."
   },
   "dependencies": {
+    "@ledgerhq/hw-transport": "^6.28.8",
```

Comment:
Shouldn't it be deps of @reef-knot/ledger-connector as concrete connector, but not general deps?

Reply:
There are imports from `@ledgerhq/hw-transport` in this package: in LedgerContext, hooks and helpers. So this package must be a dependency.
`@reef-knot/ledger-connector` is just a part, it only describes the connector. The ledger-related UI code is placed in `connect-wallet-modal` package.

Reply:
Just feels a bit weird that some of the ledger packages are required, but at the same time connector is optional, but it's ok I guess

Reply:
@kozlovzxc-at-lido I agree, and it is not okay actually, but it will be fixed later